api/src/main/java/io/strimzi/api/kafka/model/ZookeeperClusterSpec.java

```diff
@@ -96,16 +99,6 @@ public void setLogging(Logging logging) {
         this.logging = logging;
     }
 
-    @Description("TLS sidecar configuration")
-    @JsonInclude(JsonInclude.Include.NON_NULL)
-    public TlsSidecar getTlsSidecar() {
-        return tlsSidecar;
-    }
-
-    public void setTlsSidecar(TlsSidecar tlsSidecar) {
-        this.tlsSidecar = tlsSidecar;
-    }
-
```

Comment:
Removing these properties is an incompatible change. I don't _think_ that actually breaks anything on any of the versions of Kube which we support (since we're not using structural schemas), but if we were using structural schemas the existing CRs which specified this property would be rejected by the API server. So it might not cause any problems immediately until we make the CRD changes for structual schemas at which point users who still had this specified in their CRs would suddenly get errors.

Overall I think it might be safer to just leave the property in there, but mark it as `@DeprecatedProperty` and ignore it when it's set. We could remove it in `v1beta2`.